### ISSUE TITLE
Failed to compile when using with ReactJS

### ISSUE THREAD

**Issue by MikeHibbert** [ipfs-shipyard/nft.storage#289]
I installed the module using 'npm install nft.storage --save' to my ReactJS project and then ran 'npm start' to run the app and get the following error:

```
Failed to compile
./node_modules/nft.storage/src/lib.js 133:4
Module parse failed: Unexpected token (133:4)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|     const upload = transform(MAX_CONCURRENT_UPLOADS, async
|     /** @type {AsyncIterable<Uint8Array>} */
>     car => {
|       const carParts = [];
|
```
My project is written in ES7

**Comment by alanshaw** [comment]
I think this might be related to your Node.js version? What version are you running @MikeHibbert? Try v14 or ideally v16.

**Comment by MikeHibbert** [comment]
@alanshaw I'm using v14.2.0, that should be ok right?

**Comment by MikeHibbert** [comment]
Ok! update!

When I remove the /** @type {AsyncIterable<Uint8Array>} */ JSDoc from that line in src/lib.js it works fine!

**Comment by hugomrdias** [comment]
this was fixed in v3.0.2 

thank you for the issue.
